### PR TITLE
Feature/swordv2 server

### PR DESCRIPTION
**Pull request by martyn-w**
skeleton building blocks for Willow sword engine

Merge the optimise docker PR first #135 

#7 

Going to /sword/service_document (e.g. http://192.168.99.100/sword/service_document) should produce a Sword service document like this:

```
<?xml version="1.0"?>
<service xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:sword="http://purl.org/net/sword/terms/" xmlns="http://www.w3.org/2007/app">
	<sword:version>2.0</sword:version>
	<workspace collections="2">
		<collection href="TODO.url">
			<atom:title>Willow Journal of Collection 1</atom:title>
			<accept>*/*</accept>
			<accept alternate="multipart-related">*/*</accept>
			<sword:collectionPolicy>TODO: Collection Policy</sword:collectionPolicy>
			<dcterms:abstract>
This is a collection of articles from the Willow Journal of Collection 1
</dcterms:abstract>
			<sword:mediation>true</sword:mediation>
			<sword:treatment>TODO: Treatment description</sword:treatment>
			<sword:acceptPackaging>http://purl.org/net/sword/package/SimpleZip</sword:acceptPackaging>
			<sword:acceptPackaging>http://purl.org/net/sword/package/Binary</sword:acceptPackaging>
			<sword:acceptPackaging>http://purl.org/net/sword/package/METSDSpaceSIP</sword:acceptPackaging>
			<sword:service>TODO.url</sword:service>
		</collection>
		<collection href="TODO.url">
			<atom:title>Willow Journal of Collection 2</atom:title>
			<accept>*/*</accept>
			<accept alternate="multipart-related">*/*</accept>
			<sword:collectionPolicy>TODO: Collection Policy</sword:collectionPolicy>
			<dcterms:abstract>
This is a collection of articles from the Willow Journal of Collection 2
</dcterms:abstract>
			<sword:mediation>true</sword:mediation>
			<sword:treatment>TODO: Treatment description</sword:treatment>
			<sword:acceptPackaging>http://purl.org/net/sword/package/SimpleZip</sword:acceptPackaging>
			<sword:acceptPackaging>http://purl.org/net/sword/package/Binary</sword:acceptPackaging>
			<sword:acceptPackaging>http://purl.org/net/sword/package/METSDSpaceSIP</sword:acceptPackaging>
			<sword:service>TODO.url</sword:service>
		</collection>
	</workspace>
</service>

```

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,7 @@
 [submodule "geoblacklight"]
 	path = geoblacklight
 	url = https://github.com/CottageLabs/geoblacklight.git
+
+[submodule "willow_sword"]
+	path = willow_sword
+	url = git@github.com:CottageLabs/willow_sword.git
```
Comment (emanuil-tolev) [inline .gitmodules]: A server (like the demo server) will have a hard time doing `git submodule update` on a submodule that requires an SSH key to clone. Usually https (see above) is used for submodules.

Comment (emanuil-tolev) [inline .gitmodules]: We can set up the demo server for ssh access to github though. It just needs public access, and also our production environments shouldn't have this problem. Any *build* environment where the repo needs to be pulled will have it.

Comment (martyn-w) [inline .gitmodules]: HI @emanuil-tolev I've changed it from ssh:// to http://

**Comment by emanuil-tolev** [comment]
Actually, just before I merge, I'll try it locally

**Comment by martyn-w** [comment]
@emanuil-tolev make sure you init and pull the submodules

`git submodule update --init --recursive`

**Comment by emanuil-tolev** [comment]
Hahahah, schoolboy error! Quite ironic considering my earlier slack shares.

**Comment by martyn-w** [comment]
@emanuil-tolev I'm glad the irony was not lost :-)

**Comment by emanuil-tolev** [comment]
Works. I can just about review this but you'll probably need Anusha for more substantial sword-related things.

**Pull request merged**